Header header
duration dt
int32 img_width
int32 img_height
int32 feature_count
int32[] features_next  # len = 2*feature_count
int32[] features_prev
geometry_msgs/Pose relative_pose
